src/components/tooltip/CloseButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { injectIntl, IntlShape } from 'react-intl';
+
+import IconClose from '../../icon/fill/X16';
+import PlainButton from '../plain-button';
+
+// @ts-ignore flow import
+import messages from '../../common/messages';
+
+type Props = {
+    intl: IntlShape;
+    onClick: (event: React.SyntheticEvent<HTMLButtonElement, Event>) => void;
+};
+
+const CloseButton = ({ intl, onClick }: Props) => (
```

Comment:
it's out of the scope of the original issue but in the future we should look into if we could combine these two components: https://github.com/box/box-ui-elements/tree/master/src/components/close-button